enterprise/utils.py

```diff
@@ -852,3 +854,32 @@ def get_default_catalog_content_filter():
     Return default enterprise customer catalog content filter.
     """
     return settings.ENTERPRISE_CUSTOMER_CATALOG_DEFAULT_CONTENT_FILTER or DEFAULT_CATALOG_CONTENT_FILTER
+
+
+def get_request_or_stub():
```

Comment:
I think we should mention taken from edx-platform ?

Reply:
as discussed, this function should be retired if possible.